### PR TITLE
Add link to CDI Configuration

### PR DESCRIPTION
**Pull request by tevans78**
Add a link from the CDI section on beans.xml to the section about CDI configuration.

### CODE REVIEW

File: posts/2022-08-04-jakarta-core-profile-22009-beta.adoc [hunk 1]
```diff
@@ -160,7 +160,7 @@ You can control the order of multiple observer methods by using the `@Priority`
 
 === Empty beans.xml files
 
-In previous versions of CDI, an empty `beans.xml` file indicated that an archive should be treated as an explicit bean archive (the equivalent of `bean-discovery-mode="all"`). In CDI 4.0, an empty `beans.xml` now causes an archive to be treated as an implicit bean archive (the equivalent of `bean-discovery-mode="annotated"`). If necessary, you can set a configuration option to enable compatibility with previous versions.
+In previous versions of CDI, an empty `beans.xml` file indicated that an archive should be treated as an explicit bean archive (the equivalent of `bean-discovery-mode="all"`). In CDI 4.0, an empty `beans.xml` now causes an archive to be treated as an implicit bean archive (the equivalent of `bean-discovery-mode="annotated"`). If necessary, you can set a configuration option to enable compatibility with previous versions (<<cdiConfiguration, emptyBeansXmlCDI3Compatibility>>).
```
Comment (dmuelle) [inline posts/2022-08-04-jakarta-core-profile-22009-beta.adoc]:
```suggestion
In previous versions of CDI, an empty `beans.xml` file indicated that an archive should be treated as an explicit bean archive (the equivalent of `bean-discovery-mode="all"`). In CDI 4.0, an empty `beans.xml` now causes an archive to be treated as an implicit bean archive (the equivalent of `bean-discovery-mode="annotated"`). If necessary, you can <<cdiConfiguration, set the emptyBeansXmlCDI3Compatibility configuration option>> to enable compatibility with previous versions.
```

**Review by dmuelle — DISMISSED** [review]
minor suggestion

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by ReeceNana — COMMENTED** [review]
Looks Good!

**Pull request merged**